internal/impl/gcp/cloud_storage_output.go

```diff
@@ -3,6 +3,7 @@ package gcp
 import (
 	"context"
 	"fmt"
+	"path/filepath"
```

Comment:
I think you might want to use `path` instead of `path/filepath`, because `path/filepath` is OS-dependent and GCP only uses Unix-style paths.